src/sentry/static/sentry/app/components/idBadge/organizationBadge.jsx

```diff
@@ -39,3 +43,39 @@ export default class OrganizationBadge extends React.Component {
     );
   }
 }
+
+const OrganizationBadgeContainer = createReactClass({
```

Comment:
I'm undecided if we want this logic for ALL `Organization/Team-Badges`. It currently really only applies on settings pages (more specifically the breadcrumb) where you update your avatar.